net/openthread-br/Makefile

```diff
@@ -49,30 +49,26 @@ define Package/openthread-br/description
 endef
 
 define Package/openthread-br/conffiles
-/var/lib/thread
+/etc/openthread
```

Comment:
NAK. The daemon writes its state to this file, so it does not belong in /etc at all. I used /var/lib/thread because this is the default, and on by default on OpenWrt this end up on a tmpfs. According to the timestamp, this file was written to less than 6h ago. Setting the default to a path written to flash might wear flash chips. The user should explicitly decide to do that themselves if they want to.

Reply:
> Setting the default to a path written to flash might wear flash chips

That's makes sense, I was not aware of that. How do you propose then to fix failing builds e.g., https://downloads.openwrt.org/snapshots/faillogs/aarch64_cortex-a72/packages/openthread-br/compile.txt?

Reply:
Good question. I'm asking in #openwrt-devel.